.gitignore

```diff
@@ -21,4 +21,8 @@ src/burrTxt2
 src/gui/burrGui
 src/help/helpviewer
 src/unitTest
-src/config.h.in
+src/burrTxt.exe
```

Comment:
Why is config.h.in version-controlled again?  Is there a specific reason for not autogenerating it?

All the .exe files could be subsumed under `*.exe`

Reply:
It's a duplicate line, it's still git-ignored.